app/models/data.py

```diff
@@ -127,6 +124,21 @@ def pum(self):
     def fips(self):
         return fips_lookup(self.state)
 
+    @validates('totalTestResultsFieldDbColumn')
+    def validate_totalTestResultsFieldDbColumn(self, key, value):
+        """Validate the totalTestResultsFieldDbColumn value, used to calculate totalTestResults.
+
+        Acceptable values are either a valid CoreData column name or a known special keyword like 'posNeg'.
```

Comment:
where is `posNeg` coming from as a known special keyword?

Reply:
Existing behavior of the past few months. that's how the public API build tool describes it, and the Tableau stuff uses it when it looks at `covidTrackingProjectPreferredTotalTestField`, so keeping it consistent seemed to make sense.